content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -120,9 +148,9 @@ to the ambient mesh. The best part is that there was no need to restart or redep
 Send some test traffic:
 
 {{< text bash >}}
-$ kubectl exec deploy/sleep -- curl -s http://istio-ingressgateway.istio-system/productpage | head -n10
-$ kubectl exec deploy/sleep -- curl -s http://productpage:9080/ | head -n10
-$ kubectl exec deploy/notsleep -- curl -s http://productpage:9080/ | head -n10
+$ kubectl exec "$(kubectl get pod -l app=sleep -o jsonpath={.items..metadata.name})" -c sleep -- curl -s http://istio-ingressgateway.istio-system/productpage | | head -n10
```

Comment:
out of curiosity -- why replace the deploy/[deployment] with kubectl get pod?

Reply:
sorry, it is not needed...was just experimenting